pkgs/applications/science/logic/redprl/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchgit, mlton }:
+stdenv.mkDerivation {
+  name = "redprl-2016-09-22";
+  src = fetchgit {
+    url = "https://github.com/RedPRL/sml-redprl.git";
+    rev = "3215faf0d494f4ac14d6e10172329a161df192c4";
+    sha256 = "0pcq4q9xy34j7ziwbly4qxccpkcrl92r9y11bv6hdkbzwm1g2a77";
+    fetchSubmodules = true;
+  };
+  buildInputs = [ mlton ];
+  builder = builtins.toFile "builder.sh" ''
+    source $stdenv/setup
+    mkdir -p $out/bin
+    cp -r $src/* .
+    chmod -R +w src
+    chmod -R +w src
```

Comment:
Just to be sure? Double-plus write? 😋

That's a remnant from an earlier edit, thanks for catching it. I'll push a fix as soon as I can.